docs/en/ingest-management/fleet/add-fleet-server-on-prem.asciidoc

```diff
@@ -0,0 +1,173 @@
+[[add-fleet-server-on-prem]]
+= Deploy on-premises and self-managed
+
+To deploy a self-managed {fleet-server}, you need to: 
+
+* Satisfy all <<add-fleet-server-on-prem-compatibility,compatibility requirements and prerequisites>>
+* Add hosts including <<add-fleet-server-on-prem-hosts,{fleet-server} hosts and {es} hosts>>
+* <<add-fleet-server-on-prem-add-server,Add {fleet-server}>> by installing an {agent} and enrolling it in an agent policy containing the {fleet-server} integration
+
+NOTE: You can install only a single {agent} per host, which means you cannot run
+{fleet-server} and another {agent} on the same host unless you deploy a
+containerized {fleet-server}.
+
+[discrete]
+[[add-fleet-server-on-prem-compatibility]]
+== Compatibility
+
+{fleet-server} is compatible with the following Elastic products:
+
+* {stack} 7.13 or later.
+** For version compatibility: {es} >= {fleet-server} >= {agent} (except for
+bugfix releases)
+** {kib} should be on the same minor version as {es}.
+
+* {ece} 2.9 or later
++
+--
+** Requires additional wildcard domains and certificates (which normally only
+cover `*.cname`, not `*.*.cname`). This enables us to provide the URL for
+{fleet-server} of `https://.fleet.`.
+** The deployment template must contain an {integrations-server} node.
+--
++
+For more information about hosting {fleet-server} on {ece}, refer to
+{ece-ref}/ece-manage-integrations-server.html[Manage your {integrations-server}].
+
+[discrete]
+[[add-fleet-server-on-prem-prereq]]
+== Prerequisites
+
+Before setting up {fleet-server}, you will need a
+Certificate Authority (CA) certificate to configure Transport Layer Security (TLS)
+to encrypt traffic between the {fleet-server}s and {stack}.
+
+If your organization already uses the {stack}, you may already have a CA certificate. If you do not have a CA certificate, you can read more
+about generating one in <<secure-connections>>.
+
+NOTE: This is not required when testing and iterating using the *Quick start* option, but should always be used for production deployments.
+
+[discrete]
+[[add-fleet-server-on-prem-hosts]]
+== Add {fleet-server} hosts
+
+// What is it / why do you need it?
+Start by adding one or more {fleet-server} hosts.
+A {fleet-server} host is a URL your {agent}s will use to connect to a {fleet-server}.
+
+// Prereqs
+{fleet-server} hosts should meet the following requirements:
+
+* All agents can connect to the host.
+* The host also has a route to the {es} you plan to use, whether hosted by Elastic
+or deployed on-premises.
+// Need link to "minimum compute resource requirements"
+* The host meets the minimum compute resource requirements based on the maximum number
+of agents you plan to support in your deployment.
```

Comment:
>I'm not sure how to deal with this for self-managed Fleet Servers because I think we only have these figures for cloud. @nimarezainia WDYT?

@nimarezainia in your original Google doc you suggested that we "<insert link to scale compute requirements>". Do you know where that exists?

Reply:
Yes we only have figures for cloud but for now we are considering that the performance would be the same whether it's in the cloud or on-prem. At least it's the closest data we have on hand.